src/sugar3/graphics/xocolor.py

```diff
@@ -224,7 +231,14 @@ def _parse_string(color_string):
 
 
 class XoColor:
-
+    '''
+    Defines color for XO
+    
+    Args:
+        color_string (string): two html format strings separated by a comma, "white", or "insensitive"
+        If color_string is None, the user's color will be created
+        If parsed_color cannot be created, a random color will be used
```

Comment:
Can you maybe indent the ifs a little more?  They confuse sphinx:

[![https://gyazo.com/2750e04bea3ea37db914263193dec31b](https://i.gyazo.com/2750e04bea3ea37db914263193dec31b.png)](https://gyazo.com/2750e04bea3ea37db914263193dec31b)